rest/utilities_testing.go

```diff
@@ -2133,3 +2197,17 @@ func (rt *RestTester) GetSingleKeyspace() string {
 	rt.TB.Fatal("Had no collection to return a keyspace for") // should be unreachable given length check above
 	return ""
 }
+
+// getCollectionsForBLIP
+func (rt *RestTester) getCollectionsForBLIP() []string {
+	db := rt.GetDatabase()
+	var collections []string
+	for _, collection := range db.CollectionByID {
+		if base.IsDefaultCollection(collection.ScopeName(), collection.Name()) {
+			continue
```

Comment:
Should we always be skipping the default collection here?

Reply:
Not necessarily, we should probably only skip the default collection if it's the only collection.